spoor/toolchains/xcode/BUILD

```diff
@@ -42,13 +42,15 @@ py_library(
         "//spoor/toolchains/xcode/test_data/build_intermediates",
         "//spoor/toolchains/xcode/test_data/info_plists",
     ],
+    imports = ["."],
```

Comment:
Out of curiosity, why do we now need these `imports`?

Reply:
Newer **rules_python**. Without this, our imports need to be something like `from spoor.toolchains.xcode.shared import SPOOR_INSTRUMENTATION_XCODE_OUTPUT_FILE_MAP_KEY, BuildTools`, which then breaks when we place them inside of the `.xctoolchain`.